tests/benchmarks/test_arrow.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import pytest
+
+from ..utils_test import cluster_memory, timeseries_of_size, wait
+
+
+@pytest.fixture(params=[True, False], ids=["string[pyarrow]", "object"])
+def series_with_client(request, small_client):
```

Comment:
Instead of returning both the series and the client from this fixture, you can also re-use the client fixture here _and_ in the test. We do this in a few places in the benchmarks. Is there a specific reason you did not go for this pattern here?

Reply:
No good reason. I updated the fixture.